examples/push-file

```diff
@@ -3,11 +3,64 @@
 import argparse
 import json
 import logging
+import kobo.rpmlib
+import os
 
 from chexus import Client, BucketItem, TableItem
 
 LOG = logging.getLogger("push-file")
 
+def get_sigkey(rpmfile):
+    """
+    If the rpm is signed, return the sigkey.
+    rpmfile: path to an rpm
+    """
+    try:
+        rpmhead = kobo.rpmlib.get_rpm_header(rpmfile)
+    except:
+        raise RuntimeError("Cannot read RPM header: %s" % rpmfile)
+    rpmsig = kobo.rpmlib.get_keys_from_header(rpmhead).lower()
```

Comment:
If an RPM is unsigned, does get_keys_from_header return an empty string, or None?  If None, lower() will crash.

Reply:
Nice catch. Now it fails somewhat gracefully:
```
(chexus_dev) [crungeho@localhost examples]$ python push-file --debug --bucket caleigh-test --table path-to-object --file-path "/home/crungeho/exodus/rpm-4.14.2-32.el8.x86_64.rpm" --release-date "2020-02-01T05:00:00" --dryrun
Traceback (most recent call last):
  File "push-file", line 169, in <module>
    main()
  File "push-file", line 159, in main
    web_uri=cdn_path(upl_item.name, p.file_path, upl_item.checksum),
  File "push-file", line 52, in cdn_path
    get_sigkey(file_path),
  File "push-file", line 25, in get_sigkey
    raise RuntimeError("Invalid RPM signature: %s is unsigned" % rpmfile)
RuntimeError: Invalid RPM signature: /home/crungeho/exodus/rpm-4.14.2-32.el8.x86_64.rpm is unsigned
```